docs/source/references/functional.rst

```diff
@@ -308,3 +308,28 @@ retrieval_normalized_dcg [func]
 
 .. autofunction:: torchmetrics.functional.retrieval_normalized_dcg
     :noindex:
+
+
+******************
+Audio Metrics
```

Comment:
Great!
Could you move these to the top of the documentation (alphabetical order)

Reply:
You mean  something like this?
```
##############
Module metrics
##############

*************
Audio Metrics
*************

**********
Base class
**********
```

Reply:
Hello, SkafteNicki. I have moved the audio metric section on the top of `Base Classs` in the `modules.rst`. I don't know if it is what you mean... Here is the commit [add module doc](https://github.com/PyTorchLightning/metrics/pull/292/commits/1d77709fa7d24e9929d8259cbf812a1069865f64).

Reply:
[Here](https://github.com/PyTorchLightning/metrics/pull/292/commits/96c579a7726c1993e1aa02088738121b47f8f927) is the commit for `references.rst`

Reply:
Hi @quancs,
Sorry for the confusing message. The `functional.rst` file looks fine. The `module.rst` file could you move your section so it is:
- Base
- Audio
- Classification
- ect
such that everything after base is alphabetical order

Reply:
done. The commit [move Base to the top of Audio](https://github.com/PyTorchLightning/metrics/pull/292/commits/fe6e6bcf294f22fd5a94a7121d5fd179602b660e)